wordpress-mobile/gutenberg-mobile
# Invert the expect matcher values of the e2e tests

`gutenberg PR`  https://github.com/WordPress/gutenberg/pull/28770

To test:

- CI: Full E2E Test suite should pass.
- Locally: `npm run test:e2e:ios:local` and `npm run test:e2e:android:local` should work as before.

N.B I was having a local issue with Apium so I am using CI to verify the tests. If they fail I will resolve the issue. 

PR submission checklist:
